modules/julia/tutorials/julia.markdown

```diff
@@ -0,0 +1,121 @@
+
+Julia
+-------------
+Julia is a high-performance, high-level, and dynamic programming language that specializes in tasks relateted numerical, and scientefic computing. However, It can also be used for general programming with GUI and web programming. Julia can be considered as a combination of rapid interpreter style prototyping capability of Python with the raw speed of C because of its special "just-ahead-of-time" compilation.
+
+Inspite of all this, Julia severely lacks in a lot of traditional computer vision and image processing algorithms. This also hampers the usage of Julia in any pipeline that requires computer vision. The OpenCV bindings for Julia aims to solve this problem.
+
+The Bindings
+-----------------------
+The OpenCV bindings for Julia are created automatically using Python scripts at configure time and then installed with the Julia package manager on the system. These bindings cover most of the important functionality present in the core, imgproc, imgcodecs, highgui, videio, and dnn modules. These bindings depend on CxxWrap.jl and the process for usage and compilation is explained in detail below. The Bindings have been tested on Ubuntu and Mac. Windows might work but is not officially tested and supported right now.
+
+The generation process and the method by which the binding works are extremely similar to the Python bindings. The only major difference is that CxxWrap.jl does not support optional arguments. As a consequence, it's necessary to define the optional arguments in Julia code which adds a lot of additional complexity.
+
+
+How To Build The Bindings
+-----------------------
+Before you can build the bindings, make sure that you know how to build OpenCV with all the functionality you require and the contrib modules except the Julia Bindings.
+
+The pre-requisites for the Julia Bindings are:
+ - [CxxWrap.jl](https://github.com/JuliaInterop/CxxWrap.jl)
+ - [libcxxwrap-julia](https://github.com/JuliaInterop/libcxxwrap-julia)
+ - Python
+ - Julia
+
+It is recommended to use Julia 1.4+ and the latest versions of CxxWrap.jl and libcxxwrap-julia.
+
+The first step is to build [libcxxwrap-julia](https://github.com/JuliaInterop/libcxxwrap-julia) from source. The link explains how to do that. You must also setup the override in `/.julia/artifacts/Overrides.toml` as explained at the link.
+
+Once that's done you start a Julia terminal and Just start the REPL. Hit `]` and then type `add CxxWrap`.
+
+```bash
+$ julia
+...
+julia> ]
+pkg> add CxxWrap
+```
+
+This should install CxxWrap. At this step you should also check whether your libcxxwrap-julia override was set correctly or not. You can do this by checking the value of `CxxWrap.CxxWrapCore.prefix_path()` The output should show the build directory of libcxxwrap-julia
+
+```bash
+julia> using CxxWrap
+
+julia> CxxWrap.CxxWrapCore.prefix_path()
+"/home/archit/src/libcxxwrap-julia-build"
```

Comment:
please, avoid the use of absolute paths in the patches. Please, also check your other pull request that it does not contains absolute paths. Use `$HOME/` instead of `/home/archit/` in this case

Reply:
Done

Reply:
Also checked the PR3 code, no /home/archit/ directories.